runner/client/src/mill/runner/client/JansiLoader.java

```diff
@@ -0,0 +1,50 @@
+package mill.runner.client;
+
+import java.io.File;
+import java.io.IOException;
+import org.fusesource.jansi.internal.OSInfo;
+
+final class JansiLoader {
```

Comment:
Let's add a top-level comment in this file explaining what we're doing here: we're loading Jansi and extracting the necessary `.dylib` or `.jnilib`'s manually through Coursier so the extraction can be cached and we can avoid the cost of performing it every time at startup